tests/robot-tests/tests/general_public/prod_table_tool.robot

```diff
@@ -11,27 +11,27 @@ Go to 'data-tables' page
     user navigates to data tables page on public frontend
 
 Check page contains correct themes
-    user checks page for details section    Children's social care
```

Comment:
Rather than wait for every single one of these, I think it's better to wait for something that should be on the page every time, like "Choose a publication" text or the "Next step" button. I usually find if you wait for something inside the same react component as the thing you want to check, then that's good enough.

You could still add a wait for the first details section. But if the first details section is rendered, then I don't think there should be a need to wait for the others - they'll either all be rendered or none.

Reply:
This is mainly to assert that those details sections exist on the page and not to determine whether the page has loaded